posts/2020-10-23-beta-20.0.0.12.adoc

```diff
@@ -88,33 +89,32 @@ public static class BeanOne {
 }
 ----
 
-retrieves the following defined properties for the `name` and `age` fields:
+The code above retrieves the following defined properties for the `name` and `age` fields:
 
 [source, java]
 ----
 customer.name=Bob
 customer.age=24
 ----
 
-* Allows end users to easily determine which was the "winning" `ConfigSource` (the `ConfigSource` with the highest ordinal) for a specified property via link:https://github.com/eclipse/microprofile-config/blob/master/api/src/main/java/org/eclipse/microprofile/config/ConfigValue.java[ConfigValue]. A `ConfigValue` object holds information about the configuration property name, configuration value, the ConfigSource name from where the configuration property was loaded and the ordinal of the `ConfigSource`. Users can access these values with the following method:
+* Allows end users to easily determine which was the "winning" `ConfigSource` (the `ConfigSource` with the highest ordinal) for a specified property via link:https://github.com/eclipse/microprofile-config/blob/master/api/src/main/java/org/eclipse/microprofile/config/ConfigValue.java[ConfigValue]. A `ConfigValue` object holds information about the configuration property name, configuration value, the ConfigSource name from where the configuration property was loaded and the ordinal of the `ConfigSource`. You can access these values with the following method:
```

Comment:
Should this be "end users" or should it be changed to "you" ? @lauracowen

Reply:
Fixed